### ISSUE TITLE
Feature request: Automatic names extraction

### ISSUE THREAD

**Issue by WilliamDelamare** [maximedaniel/ESTIA-TechnoWebValidator#1]
Could the script considers the excel or pdf files extracted from Pegasus instead of hardcoding names?

**Comment by maximedaniel** [comment]
Yes, of course. I was not aware that PEGASUS can extract student lists as PDF or EXCEL. I will check it out.
